{"priority": "major", "kind": "bug", "repository": {"links": {"self": {"href": "data/repositories/osrf/gazebo.json"}, "html": {"href": "#!/osrf/gazebo"}, "avatar": {"href": "data/bytebucket.org/ravatar/{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}ts=1694483"}}, "type": "repository", "name": "gazebo", "full_name": "osrf/gazebo", "uuid": "{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}"}, "links": {"attachments": {"href": "data/repositories/osrf/gazebo/issues/1342/attachments_page=1.json"}, "self": {"href": "data/repositories/osrf/gazebo/issues/1342.json"}, "watch": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/gazebo/issues/1342/watch"}, "comments": {"href": "data/repositories/osrf/gazebo/issues/1342/comments_page=1.json"}, "html": {"href": "#!/osrf/gazebo/issues/1342/system-fails-to-run-all-of-a-sudden-on"}, "vote": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/gazebo/issues/1342/vote"}}, "reporter": {"display_name": "James Taylor", "uuid": "{80a5efd1-140d-4624-a75d-d114628c1db8}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B80a5efd1-140d-4624-a75d-d114628c1db8%7D"}, "html": {"href": "https://bitbucket.org/%7B80a5efd1-140d-4624-a75d-d114628c1db8%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/3cbda94bb60d047f7d10c50d7d847d5ed=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsJT-3.png"}}, "nickname": "jrtaylor", "type": "user", "account_id": "557058:e8ac7e8b-59fe-4119-a528-7d0f5b851e6e"}, "title": "System fails to run all of a sudden (on file that ran fine moments before) from seemingly outside (packages) cause.", "component": null, "votes": 0, "watches": 2, "content": {"raw": "Was testing in Gazebo fine when started receiving exceptions that my model file did not parse properly.  The model file was unchanged from instance to instance and the underlying support code I utilized to validate was unchanged.  After double and triple checking that these files were stable (including redownloading a guaranteed working copy from my repo), I decided to reboot.  Upon beginning to reboot, I noted that dart had attempted to push an update today (4.2), but failed due to my having 4.1 and the system would not integrate dart-core4.2 with 4.1 on the machine.  After reboot, I continued having the same problem with my models not parsing.  Decided to try to reinstall gazebo, but it now failed to cmake because system did not have dart 4.2 (note, I did not pull gazebo this all changed in the background).  Wiped dart (which wiped libccd) and pulled 4.2 which pulled libccd2.  Now on make, ld fails because I no longer have libccd.  \r\n\r\nDo you @nkoenig or @hsu have any advice to get me running again?  I am gimped at the moment.  My next step is to wipe gazebo, pull and start over.  Not sure why the dart push (or other general push) killed my system, but it somehow did.", "markup": "markdown", "html": "<p>Was testing in Gazebo fine when started receiving exceptions that my model file did not parse properly.  The model file was unchanged from instance to instance and the underlying support code I utilized to validate was unchanged.  After double and triple checking that these files were stable (including redownloading a guaranteed working copy from my repo), I decided to reboot.  Upon beginning to reboot, I noted that dart had attempted to push an update today (4.2), but failed due to my having 4.1 and the system would not integrate dart-core4.2 with 4.1 on the machine.  After reboot, I continued having the same problem with my models not parsing.  Decided to try to reinstall gazebo, but it now failed to cmake because system did not have dart 4.2 (note, I did not pull gazebo this all changed in the background).  Wiped dart (which wiped libccd) and pulled 4.2 which pulled libccd2.  Now on make, ld fails because I no longer have libccd.  </p>\n<p>Do you @nkoenig or @hsu have any advice to get me running again?  I am gimped at the moment.  My next step is to wipe gazebo, pull and start over.  Not sure why the dart push (or other general push) killed my system, but it somehow did.</p>", "type": "rendered"}, "assignee": {"display_name": "John Hsu", "uuid": "{0a186eae-abf0-4514-a951-23db5eccc286}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B0a186eae-abf0-4514-a951-23db5eccc286%7D"}, "html": {"href": "https://bitbucket.org/%7B0a186eae-abf0-4514-a951-23db5eccc286%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:f3968cd3-4910-4384-8349-482a6c7889ec/5445ce6e-6273-47f0-84eb-621c86ca11cb/128"}}, "nickname": "hsu", "type": "user", "account_id": "557058:f3968cd3-4910-4384-8349-482a6c7889ec"}, "state": "new", "version": {"name": "all", "links": {"self": {"href": "data/repositories/osrf/gazebo/versions/225575.json"}}}, "edited_on": null, "created_on": "2014-11-26T00:32:21.269219+00:00", "milestone": null, "updated_on": "2016-09-14T19:22:04.186806+00:00", "type": "issue", "id": 1342}